README.md

```diff
@@ -122,6 +122,8 @@ These settings are specific to VSCodeVim.
 | vim.substituteGlobalFlag         | Similar to Vim's `gdefault` setting. `/g` flag in a substitute command replaces all occurrences in the line. Without this flag, replacement occurs only for the first occurrence in each line. With this setting enabled, the `g` is on by default.                                                                                                                                                                              | Boolean | false                    |
 | vim.useCtrlKeys                  | Enable Vim ctrl keys overriding common VS Code operations such as copy, paste, find, etc.                                                                                                                                                                                                                                                                                                                                        | Boolean | true                     |
 | vim.visualstar                   | In visual mode, start a search with `*` or `#` using the current selection                                                                                                                                                                                                                                                                                                                                                       | Boolean | false                    |
+| vim.yankHighlighting             | Enable highlighting when yanking                                                                                                                                                                                                                                                                                                                                                                                                 | Boolean | false                    |
```

Comment:
Can we add these under their own sections?

```
vim.highlightedyank.enable
vim.highlightedyank.color
vim.highlightedyank.duration
```

You can see `vim.debug` for reference.